charts/consul/values.yaml

```diff
@@ -1034,6 +1080,9 @@ externalServers:
   # The HTTPS port of the Consul servers.
   httpsPort: 8501
 
+  # The GRPC port of the Consul servers.
+  grpcPort: 8503
+
```

Comment:
between now and GA of this feature, @ishustava, you and I and the team should have a conversation about potentially swapping out the client to use 8503 and the servers to use 8502 for GRPC given agentless should ensure no competition for port, in which case I would like the helm default to be the same as the Consul defaults. I have kinda led us down the 8503 track and would like to get us to a feasible long term solution.

Reply:
That sounds like a good plan!